plugins/buddy-plugin-postcss/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2016 Pope-Industries &lt;alex@pope-industries.com&gt;
```

Comment:
By the way, I’ve added some plugins that I think are "safe defaults". No rush to merge this, though.